base.nix

```diff
@@ -142,7 +142,7 @@ with lib;
       unset activationScript
     '';
     # nix-build -A squashfs && ls -lLh result
-    system.build.squashfs = pkgs.callPackage <nixpkgs/nixos/lib/make-squashfs.nix> {
+    system.build.squashfs = pkgs.callPackage (pkgs.path + "/nixos/lib/make-squashfs.nix") {
```

Comment:
I noticed that because my pinned nixpkgs/nixos was no longer compatible with the squashfs from pkgs.